README.rst

```diff
@@ -40,21 +40,32 @@ In order to install the package in the current environment:
 
     maturin develop
 
-Build and Release:
+Release
+=======
+
+Update version in ``Cargo.toml`` and:
 
 .. code-block ::
 
     vim Cargo.toml
     git ci -am "Bump version"
     git tag -a v1.2.3
+    git push v1.2.3
 
-Update version in ``Cargo.toml`` and:
+Publish wheel for your host OS:
 
 .. code-block ::
 
     maturin build
     maturin publish
 
+
+Publish wheels of all architectures on PyPi:
```

Comment:
It would be nice if this was handed in the build action so that no manual steps were needed. Maybe something to tackle in the future.